.github/workflows/trigger-workflow.yml

```
@@ -1,9 +1,18 @@
 name: Trigger agent manager
 
 on:
-  push:
-    branches:
-      - 'dev'
+  workflow_call:
+    inputs:
+      event_type:
+        description: 'Event type to send to the agent manager'
+        required: true
+        type: choice
```

actionlint (1.7.8)

invalid value "choice" for input type of workflow_call event. it must be one of "boolean", "number", or "string"

(syntax-check)